Mathlib/Order/WithBot.lean

```diff
@@ -920,6 +920,18 @@ instance noMinOrder [LT α] [NoMinOrder α] [Nonempty α] : NoMinOrder (WithTop
 
 end WithTop
 
+section WithBotWithTop
+
+lemma WithBot.forall_ge_iff_eq_top [Preorder α] [Nonempty α] [NoMaxOrder α]
```

Comment:
Maybe `simp`?

Reply:
It cannot be because the discr_tree simp key is bad.